changelogs/fragments/20250421-remove_k8s_inventory_plugin.yaml

```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
```

Comment:
Is this the correct category of change? Should it be `breaking_changes` or `minor_changes` instead?

Reply:
The change type here is `breaking_changes`